app/controllers/version_controller.rb

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2015 The Regents of the University of Michigan.
```

Comment:
So my institution requires me to insert these headers on the files I write, but I didn't write this file.  So you're free to not include it.